<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Silkscreen</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dot Matrix: Game Boy Emulator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource.qrc">
    <normaloff>:/assets/icons/icon.ico</normaloff>:/assets/icons/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="screen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>576</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>576</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>576</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource.qrc">:/assets/imgs/bg_new.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>26</height>
    </size>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Silkscreen</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenROM"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEmulation">
    <property name="font">
     <font>
      <family>Silkscreen</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Emulation</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionDmgMode"/>
    <addaction name="actionShowBootScreen"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="font">
     <font>
      <family>Silkscreen</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuScale">
     <property name="font">
      <font>
       <family>Silkscreen</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Scale</string>
     </property>
     <addaction name="action0_5x"/>
     <addaction name="action1x"/>
     <addaction name="action1_5x"/>
     <addaction name="action2x"/>
    </widget>
    <widget class="QMenu" name="menuPalette">
     <property name="font">
      <font>
       <family>Silkscreen</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string>DMG Palette</string>
     </property>
     <widget class="QMenu" name="menuSGB">
      <property name="font">
       <font>
        <family>Silkscreen</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Super Game Boy</string>
      </property>
     </widget>
     <addaction name="menuSGB"/>
    </widget>
    <addaction name="menuScale"/>
    <addaction name="menuPalette"/>
    <addaction name="separator"/>
    <addaction name="actionShowBackground"/>
    <addaction name="actionShowWindow"/>
    <addaction name="actionShowSprites"/>
   </widget>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="actionKeyBindings"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionEnableLogging"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmulation"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuControls"/>
   <addaction name="menuDebug"/>
  </widget>
  <action name="actionOpenROM">
   <property name="text">
    <string>Open ROM</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
    </font>
   </property>
  </action>
  <action name="action0_5x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0.5x</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionKeyBindings">
   <property name="text">
    <string>Key Bindings</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionShowBootScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Boot Screen</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
    </font>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionShowBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Background</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionShowWindow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Window</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionShowSprites">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Sprites</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionEnableLogging">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Logging</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionDmgMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DMG Mode</string>
   </property>
  </action>
  <action name="action1x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1x</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action1_5x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.5x</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action2x">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2x</string>
   </property>
   <property name="font">
    <font>
     <family>Silkscreen</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="actionWho_did_this">
   <property name="text">
    <string>Who did this?</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
